Stock_Alert/errorhandling.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+
+class ErrorHandling:
+    def __init__(self, endpoint):
+        self.endpoint = endpoint
+
+
+    def get_hold_of(self, **kwargs):
```

Comment:
Simply adding params an argument would work here. Simplicity is beautiful, and you don't need kwargs for now.
```suggestion
    def make(self, params):
```

Reply:
Changed it according to your suggestion.